src/blueprints/tv2/helpers/tv2-studio-blueprint-configuration-mapper.ts

```diff
@@ -157,6 +165,18 @@ export class Tv2StudioBlueprintConfigurationMapper {
     })
   }
 
+  private mapToVideoMixerType(videoMixerTypeString: string): VideoMixerType {
+    switch(videoMixerTypeString) {
```

Comment:
The `String` suffix seems redundant. If you want to make it explicit that it is no of type `VideoMixerType`, then either `coreVideoMixerType` or `rawVideoMixerType` are some options :)